config/locales/en.yml

```diff
@@ -1836,6 +1839,13 @@ en:
       group_error: There group has errors
       numeric_operator_error: can only be used on numbers
       uniqueness_error: must be unique
+    workflow_execution_samplesheet_params_validator:
+      attachment_format_error: property '%{property}' must match the expected file format
```

Comment:
We should include the filename and expected value to these errors, it is hard to deduce which file is causing the problem when we have many samples/files.

Reply:
Agreed that we should include the file name and expected values to these errors. Otherwise pr works great

Reply:
Error message updated in 2dfb89f, I'm going to leave out display of the filename as that doesn't actually help map back to the `samplesWorkflowExecutionsAttributes` but I have updated it to include regex that is being applied. The index in the error message refers to the position in the array of `samplesWorkflowExecutionsAttributes`. So programmatically this can be easily mapped back.